cpu/kinetis/periph/i2c.c

```diff
@@ -36,13 +36,13 @@
 #include "periph_conf.h"
 #include "periph/i2c.h"
 
-#define ENABLE_DEBUG    (0)
+#define ENABLE_DEBUG        0
 /* Define ENABLE_TRACE to 1 to enable printing of all TX/RX bytes to UART for extra verbose debugging */
-#define ENABLE_TRACE    (1)
+#define ENABLE_TRACE        (1)
 /* Define ENABLE_INIT_DEBUG to 1 to enable DEBUG prints in i2c_init. Currently
  * this causes the system to hang when running i2c_init during boot because of
  * uninitialized stdio UART */
```

Comment:
Out of scope, but I think this is no longer the case. I don't have any kinetis board to confirm, maybe somebody else could check this? (But please after this PR is merged, to avoid merge conflicts.)